openapi.yaml

```diff
@@ -1,190 +1,514 @@
 swagger: '2.0'
 info:
-  title: Express API Skeleton
-  description: Skeleton for Express APIs
+  title: Corvallis Parks API
+  description: an api for the parks of Corvallis, OR
   version: v1
   license:
     name: GNU Affero General Public License Version 3
     url: http://www.gnu.org/licenses/agpl-3.0.en.html
   contact:
-    name: IS Data Architecture Team
-    url: https://is.oregonstate.edu/data-architecture
-    email: isdataarchitecture@oregonstate.edu
+    name: David Kalcic
+    email: kalcicd@oregonstate.edu
 schemes:
   - https
 host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/parks-api
+consumes:
+  - application/json
 produces:
   - application/json
 paths:
-  /pets:
+  /parks:
     get:
-      summary: List all pets
+      summary: List all parks
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - parks
+      description: Get all parks
+      operationId: getParks
       parameters:
-        - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
-        - in: query
-          name: species
-          description: Search by pet species
-          required: false
-          type: string
+        - $ref: '#/parameters/filterAmenitiesAll'
+        - $ref: '#/parameters/filterAmenitiesSome'
+        - $ref: '#/parameters/filterName'
+        - $ref: '#/parameters/filterCity'
+        - $ref: '#/parameters/filterState'
+        - $ref: '#/parameters/filterZip'
       responses:
         '200':
-          description: Successful response
+          description: Successful Response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/ParksResult'
         '400':
-          description: Bad request
+          $ref: '#/responses/400'
+        '500':
+          $ref: '#/responses/500'
+    post:
+      summary: Creates a new park
+      tags:
+        - parks
+      operationId: createPark
+      parameters:
+      - $ref: '#/parameters/parkPostBody'
+      responses:
+        '201':
+          description: Successfully created new park
+          headers:
+            Location:
+              type: string
+              description: Location of newly created park
```

Comment:
for the description of these Location headers, would you mind somehow specifying that it is the resource location (ie a link to the resource)? This could avoid confusion with the "location" of the newly created park that is an attribute of the park object.